src/HomeAutio.Mqtt.Core/BrokerSettings.cs

```diff
@@ -29,5 +29,10 @@ public class BrokerSettings
         /// Broker reconnect delay in seconds, default 5.
         /// </summary>
         public int BrokerReconnectDelay { get; set; } = 5;
+
+        /// <summary>
+        /// Whether to use TLS for the connection or not. Defaults to false.
+        /// </summary>
+        public bool UseTls {get; set; } = false;
```

Comment:
Should follow the "Brokeru" prefix the rest of the settings use here, so BrokerUseTls would be fine. Also you are missing a space between the "{" and "get" which is failing the build.

Reply:
Sorry, I did that late last night and totally missed that pattern 🤦‍♂️